modules/nw-mutual-tls-auth.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id=nw-mutual-tls-auth_{context}]
+= Configuring Mutual TLS Authentication
+
+You can configure the Ingress Controller to enable mutual TLS (mTLS) authentication by setting a `spec.clientTLS` value. The `clientTLS` value configures the Ingress Controller to verify client certificates. This configuration includes setting a `clientCA` value, which is a reference to a config map. The config map contains the PEM-encoded CA certificate bundle that is used to verify a client's certificate. Optionally, you can configure a list of certificate subject filters.
+
+If the `clientCA` value specifies an X509v3 certificate revocation list (CRL) distribution point, the Ingress Operator downloads the CRL and configures the Ingress Controller to acknowledge it. Requests that do not provide valid certificates are rejected.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+Configure Mutual TLS authentication using the `spec.ClientTLS` value for the Ingress Controller.
+
+. Create a ConfigMap that is in the `openshift-config` namespace:
++
+[source,terminal]
+----
+$ oc create configmap router-ca-certs-default --from-file=ca-bundle.pem=client-ca.crt -n openshift-config
+----
++
+[NOTE]
+====
+The config map data key must be `ca-bundle.pem`, and the data value must be a CA certificate in PEM format.
+====
+
+. Edit the `IngressController` resource in the `openshift-ingress-operator` project:
++
+[source,terminal]
+----
+$ oc edit IngressController default -n openshift-ingress-operator
+----
+
+. Edit the yaml:
```

Comment:
Maybe ". Add the `spec.clientTLS` field and subfields to configure mutual TLS:"

If you prefer to keep what you have, consider replacing "yaml" with either "the manifest" or "YAML".